app/js/components/identifiers/addIdentifier.jsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import PropTypes from 'react-proptypes';
+import apiCall from '../../utilities/apiHelper';
+
+class NewIdentifier extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            newState: false,
```

Comment:
Any reason why this variable is called `newState`?

Reply:
Meant for when a new identifier is being created but I see the confusion in the naming.